.ci/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -37,6 +37,11 @@ call %INSTALLER_DIR%\activate_miniconda3.bat
 if errorlevel 1 goto fail
 if not errorlevel 0 goto fail
 
+:: Update CMake
+call choco upgrade -y cmake --no-progress --installargs 'ADD_CMAKE_TO_PATH=System' --apply-install-arguments-to-dependencies --version=3.27.9
```

Comment:
Isn't linux on 4.0.0 ? Should we upgrade all the way there while we're at it?

Reply:
I tried 3.28, there are some breaking changes make CI fail...